rs/sns/governance/src/types.rs

```diff
@@ -369,6 +369,12 @@ impl NervousSystemParameters {
     /// hosting the SNS.
     pub const MAX_NUMBER_OF_PRINCIPALS_PER_NEURON_CEILING: u64 = 15;
 
+    /// This is a lower bound for `max_dissolve_delay_seconds`.
+    /// Decreasing it below this number is problematic because other canisters
+    /// assumes that there are allowed to be at least 5 principals per neuron
+    /// when setting up the neurons from the swap.
+    pub const MAX_NUMBER_OF_PRINCIPALS_PER_NEURON_FLOOR: u64 = 5;
```

Comment:
Note: 5 is currently the default and no SNSs have lowered it below 5 last I checked